forgae-shape-vue-command.md

```diff
@@ -0,0 +1,239 @@
+# TUTORIAL: Forgae has made it simple to build Æpps with the front-end framework
+## Tutorial Overview
+This tutorial will teach you how to to create an aepp with Vue.js front-end framework and **forgae** integration. You will learn how to:
+- Use the **forgae** shape command;
+- Communicate between the frontend(SPA with Vue.js) and the Sophia smart contract;
+
+## Prerequisites
+- Installed the **forgae** framework (take a look over [installing forgae](https://dev.aepps.com/tutorials/smart-contract-deployment-in-forgae.html) section)
+- Some familiarity with the **forgae** framework and development of Sophia smart contracts. If you are not there yet, we recommend checking some of these [development tutorials](https://dev.aepps.com/tutorials/README.html).
+- Installed **git** ([installing git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git))
+
+## Plan
+**Forgae** has made it simple to build Vue.js based Æpps by developing pre-configured boilerplate of a project called **forgae shape**.
+This tutorial will lead you throughout process of creating such Aepp.
+
+## Forgae shape command
+```
+forgae shape [name]
```

Comment:
got stuck here, what is `name`?

was clear after reading the bottom, but somehow needs some clarification.
also this indicated `vue` is not the only option

Reply:
```Vue``` is the only option till now, but we are planning to add react and angular shapes too.